llvm/test/Transforms/InstCombine/shift-logic.ll

```diff
@@ -240,6 +240,21 @@ define i32 @lshr_or_extra_use(i32 %x, i32 %y, ptr %p) {
   ret i32 %sh1
 }
 
+define i32 @lshr_or_extra_use_multiuse(i32 %x, ptr %p) {
```

Comment:
Ah sorry, I now see that the lshr_or_extra_use test has the multi-use on the binop, not the shift. So please make this `store i32 %sh0, ptr %p` and then you'll have to add a second test that's the same just with the variable `%y` operand in the or.

Reply:
Done!